.php_cs.dist

```diff
@@ -36,6 +36,7 @@ return PhpCsFixer\Config::create()
         'simplified_null_return' => true,
         'strict_comparison' => true,
         'strict_param' => true,
+        'yoda_style' => false,
```

Comment:
Believe it or not, yoda_style = true is a rule that exists now on php-cs-fixer, and is enabled by default.

Reply:
Yoda style is not enabled by default, it's a Symfony coding standard issue. It should be kept id you want to follow this standard.

Reply:
If you do not specify yoda_style = false, cs-fixer will yodaify the whole project. It must be the default now.

Reply:
@luispabon yeah, *because* you require the `@Symfony` rule set and yoda style is one of the rule of the Symfony coding standard.

https://github.com/FriendsOfPHP/PHP-CS-Fixer

> yoda_style [@Symfony]